docs/containers-transports.5.md

```diff
@@ -0,0 +1,53 @@
+% CONTAINERS-TRANSPORTS(5) Containers Transports Man Page
+% Valentin Rothberg
+% April 2019
+
+# NAME
+containers-transports - description of supported transports for copying container images
+
+# DESCRIPTION
+
+**containers-storage:**_docker-reference_
+
+An image located in a local containers storage image store.
+Please refer to containers-storage.conf(5) for further information.
+
+**dir:**_path_
+
+An existing local directory _path_ storing the manifest, layer tarballs and signatures as individual files.
+This is a non-standardized format, primarily useful for debugging or noninvasive container inspection.
+
+**docker://**_docker-reference_
+
+An image in a registry implementing the "Docker Registry HTTP API V2".
+By default, uses the authorization state in either `$XDG_RUNTIME_DIR/containers/auth.json`, which is set using `(podman login)`.
+If the authorization state is not found there, `$HOME/.docker/config.json` is checked, which is set using `(docker login)`.
+
+**docker-archive:**_path_[**:**_docker-reference_]
+
+An image is stored in the `docker save` formatted file.
+_docker-reference_ is only used when creating such a file, and it must not contain a digest.
+Note that it is possible to copy data to stdin by specifying `docker-archive:/dev/stdin`.
+
+**docker-daemon:**_docker-reference_
+
+An image _docker-reference_ stored in the docker daemon internal storage.
+_docker-reference_ must contain either a tag or a digest.
+Alternatively, when reading images, the format can be docker-daemon:algo:digest (an image ID).
+
+**oci:**_path_**:**_tag_
+
+An image _tag_ in a directory compliant with "Open Container Image Layout Specification" at _path_.
+
+**ostree:**_image_[**@**_/absolute/repo/path_]
+
+An image in local OSTree repository.
+_/absolute/repo/path_ defaults to _/ostree/repo_.
+
+## SEE ALSO
```

Comment:
Should we have some examples using skopeo and podman?

This man page is going to be rather confusing without some examples/use cases.

Reply:
Sure, we could add an #EXAMPLES section showing some examples with podman, buildah and skopeo.

Reply:
This is a reference of the details; let’s not turn this into a copy/pull/push tutorial. Sure, let’s add links to `skopeo-copy(1)` and `podman-pull(1)` here; but examples, if any, should primarily show the references, not the tools.

Reply:
Right I would just like to see examples, probably in skopeo,

Copying from one registry to another,
Copying from a Docker-daemon to a oci image directory
Copying from a registry into container storage.

I don't care what tool you do it with.  To me examples speak much better then words.

Reply:
Done.